core/converters.go

```diff
@@ -60,3 +62,37 @@ func CalculateHash(
 	hash := hasher.Compute(string(mrsData))
 	return hash, nil
 }
+
+func plural(count int, singular string) (result string) {
+	if count < 2 {
+		result = strconv.Itoa(count) + " " + singular + " "
+	} else {
+		result = strconv.Itoa(count) + " " + singular + "s "
+	}
+	return
+}
+
+// SecondsToHumanFormat transform seconds input in a human friendly format
+func SecondsToHumanFormat(input int) string {
```

Comment:
This is not tested. Add some tests with some edge cases.

Reply:
Done.